### ISSUE TITLE
http://proxy.man/ssl, the sever cannot be found

### ISSUE THREAD

**Issue by thepinyo** [ProxymanApp/Proxyman#218]
### Proxyman version? (Ex. Proxyman 1.4.3)
: 1.4.5 (1450)

### macOS Version? (Ex. mac 10.14)
10.14.3

### Steps to reproduce 
open safari on iphone xs(MT9F2TH/A), ios 12.0(16A366)
open url http://proxy.man/ssl

### Expected behavior
download cer

**Comment by NghiaTranUIT** [comment]
Hey @thepinyo , it's not a real website. It's a local website, which severs by local Proxyman' server.

So, if you're opening Proxyman, and config Wifi Proxy in iPhone to Proxyman. You can open proxy.man/ssl in Safari app.

Here is quick tutorial: https://proxyman.io/blog/2019/06/How-I-use-Proxyman-to-see-HTTP-requests-responses-on-my-iPhone.html

**Comment by NghiaTranUIT** [comment]
Btw, I will update the explanation on iOS Guideline windows, so everyone could be aware 👍

**Comment by thepinyo** [comment]
@NghiaTranUIT yes, i test on iphone xs it's work fine but on iphone 6 plus it can't download cer.

**Comment by NghiaTranUIT** [comment]
@thepinyo  what's OS version of your ip 6+?

**Comment by thepinyo** [comment]
@NghiaTranUIT  iphone 6 Plus(MGA92TH/A), ios 12.1(16B92)

**Comment by NghiaTranUIT** [comment]
I've upgraded how the local server http://proxy.man/ssl works, and will officially release tmr. Stay tuned 🎉

**Comment by debian109** [comment]
Hi @NghiaTranUIT , I can not access http://proxy.man/ssl in android phone. Could you change to https... ?

**Comment by NghiaTranUIT** [comment]
Hey, `http://proxy.man/ssl` is an endpoint in a local server (Proxyman app), not a real endpoint, so there is no reason to support HTTPS.

If you are not able to connect to `http://proxy.man/ssl`, please check out step 3 in this blog: https://docs.proxyman.io/troubleshooting/my-ios-devices-couldnt-connect-to-proxyman-via-proxy

Your macbook might reject the incoming connection from your Android devices
